README.md

```diff
@@ -1 +1,1146 @@
+};
+
+@kubernetes:Ingress {
+    hostname:"ballerina.guides.io",
+    name:"message_transformation",
+    path:"/"
+}
+@kubernetes:Service {
+    serviceType:"NodePort",
+    name:"contentfilter"
+}
+@kubernetes:Service {
+    serviceType:"NodePort",
+    name:"validator"
+}
+@kubernetes:Service {
+    serviceType:"NodePort",
+    name:"enricher"
+}
+@kubernetes:Service {
+    serviceType:"NodePort",
+    name:"backend"
+}
+@kubernetes:Deployment {
+    image:"ballerina.guides.io/message_transformation_service:v1.0",
+    name:"ballerina-guides-message-transformation-service",
+    baseImage:"ballerina/ballerina-platform:0.980.0",
+    copyFiles:[{target:"/ballerina/runtime/bre/lib",
+        source:<path_to_JDBC_jar>}]
+}
+
+endpoint http:Listener contentfilterEP {
+    port:9090
+};
+endpoint http:Listener claimvaditadeEP {
+    port:9094
+};
+endpoint http:Listener contentenricherEP {
+    port:9092
+};
+endpoint http:Listener backendEP {
+    port:9093
+};
+//define endpoints for services
+endpoint http:Client validatorEP {
+    url: "http://localhost:9094/validator"
+};
+endpoint http:Client enricherEP {
+    url: "http://localhost:9092/enricher"
+};
+endpoint http:Client clientEP {
+    url: "http://localhost:9093/backend"
+};
+
+//define the global variables
+public json payload1;
+public json payload2;
+
+//service for the content filter pattern
+service<http:Service> contentfilter bind contentfilterEP {
+.
+.
+}
+//the student ID validator service
+service<http:Service> validator bind claimvaditadeEP {
+.
+.
+}
+//The content enricher service
+service<http:Service> enricher bind contentenricherEP {
+.
+.
+}
+//client endpoint service to display the request payload
+service<http:Service> backend bind backendEP {
+.
+.
+}
+```
+- Here we have used ``  @kubernetes:Deployment `` to specify the docker image name which will be created as part of building this service. `copyFiles` field is used to copy the MySQL jar file into the ballerina bre/lib folder. Make sure to replace the `<path_to_JDBC_jar>` with your JDBC jar's path.
+- Please note that if you are using minikube it is required to add the `` dockerHost `` and `` dockerCertPath `` configurations under ``  @kubernetes:Deployment ``.
+eg:
+``` ballerina
+@kubernetes:Deployment {
+    image:"ballerina.guides.io/message_transformation_service:v1.0",
+    name:"ballerina-guides-message-transformation-service",
+    baseImage:"ballerina/ballerina-platform:0.980.0",
+    copyFiles:[{target:"/ballerina/runtime/bre/lib",
+        source:<path_to_JDBC_jar>}]
+    dockerHost:"tcp://<MINIKUBE_IP>:<DOCKER_PORT>",
+    dockerCertPath:"<MINIKUBE_CERT_PATH>"
+}
+```
+
+- We have also specified `` @kubernetes:Service `` so that it will create a Kubernetes service which will expose the Ballerina service that is running on a Pod.  
+- In addition, we have used `` @kubernetes:Ingress `` which is the external interface to access your service (with path `` /`` and hostname ``ballerina.guides.io``)
```

Comment:
Use single backticks in these instances. Apply elsewhere if applicable.

Reply:
Done